frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -325,133 +404,183 @@ export const OperandList: React.FC<OperandListProps> = (props) => {
       data={data}
       EmptyMsg={EmptyMsg}
```

Comment:
I'd love to move the nested EmtpyMsg component out of this component as something like "DefaultEmptyMessage" so that we aren't declaring a component inside another component. We also probably need to accept a prop that can override the default empty message:
```suggestion
      EmptyMsg={props.EmptyMsg ?? DefaultEmptyMsg}
```